L/libsndfile/build_tarballs.jl

```diff
@@ -47,4 +47,4 @@ dependencies = [
 ]
 
 # Build the tarballs, and possibly a `build.jl` as well.
-build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies)
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6")
```

Comment:
@staticfloat I can never remember if we need a minor version bump or if a patch release will work as well?

Reply:
patch release works.  Just not build number.